arbitrary-users-patch/base_images

```diff
@@ -10,3 +10,4 @@ che-nodejs8-centos      registry.centos.org/che-stacks/centos-nodejs
 che-php-7               eclipse/php:7.1-che7
 che-python-3.6          centos/python-36-centos7:1
 che-python-3.7          python:3.7.4-slim
+che-qurkus-19.2.1       quay.io/quarkus/centos-quarkus-maven:19.2.1
```

Comment:
```suggestion
che-quarkus-19.2.1       quay.io/quarkus/centos-quarkus-maven:19.2.1
```

Reply:
Additionally, consider just using `che-quarkus` or `che-quarkus-19`; if we name it after a bugfix release we will have to start a new repo for `19.2.2`, etc.

I'd prefer the images we push to quay.io to refer to compatible sets of versions, rather than specific targetted versions.

Reply:
I'v changed it to **che-quarkus**. 19.2.1 - it is version of GraalVM, so probably for now it makes sense to name it che-quarkus.

Reply:
:+1: